memgpt/local_llm/chat_completion_proxy.py

```diff
@@ -39,21 +43,41 @@ def get_chat_completion(
         llm_wrapper = simple_summary_wrapper.SimpleSummaryWrapper()
     elif model == "airoboros-l2-70b-2.1":
         llm_wrapper = airoboros.Airoboros21InnerMonologueWrapper()
+    elif model == "airoboros-l2-70b-2.1-grammar":
+        llm_wrapper = airoboros.Airoboros21InnerMonologueWrapper(include_opening_brace_in_prefix=False)
+        # grammar_name = "json"

```

Comment:
stray comments?

Reply:
Imma just leave this in there to indicate that there's an alt grammar, will strip at a later point